src/libraries/PositionConfig.sol

```diff
@@ -24,7 +68,7 @@ library PositionConfigLibrary {
             mstore(add(fmp, 0x14), calldataload(add(config, 0x20))) // currency1: [0x20, 0x34)
             mstore(fmp, calldataload(config)) // currency0: [0x0c, 0x20)
 
-            id := keccak256(add(fmp, 0x0c), 0x48) // len is 72 bytes
+            id := shr(1, keccak256(add(fmp, 0x0c), 0x48)) // len is 72 bytes, truncate upper bit of the hash
```

Comment:
re the comment - technically it truncates the LOWER bit of the hash because it shifts right no?
if a hash was `10110011` and you shift right one it becomes `01011001` so youve lost the lower bit of the hash. And the _upper_ bit is guaranteed to be 0

Reply:
Right, yes, the bit that is signalling is the "upper" bit, I agree.
But the bit that is truncated off the hash is the lowest bit of the hash.

So "truncate upper bit of the hash" is incorrect

Reply:
I dont mind "upper" i just think the comment is wrong

Reply:
ohh I see what you mean! yes that makes sense